### PR TITLE
Use pathname.startsWith to detect accelerator route

### PR DESCRIPTION
**Pull request by karthikbtf**
- this still doesn't render people and deliverables not sure why

### CODE REVIEW

**Review by constanzauanini — APPROVED** [review]
Thanks!

File: src/hooks/useAcceleratorConsole.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 import { APP_PATHS } from 'src/constants';
 import { useUser } from 'src/providers';
 import { isAllowed } from 'src/utils/acl';
 
 const useAcceleratorConsole = () => {
-  const isAcceleratorRoute = !!useMatch(APP_PATHS.ACCELERATOR);
   const { user } = useUser();
+  const location = useLocation();
+  const isAcceleratorRoute = location.pathname.startsWith(APP_PATHS.ACCELERATOR);
```
Comment (constanzauanini) [inline src/hooks/useAcceleratorConsole.ts:10]: I just checked and we can do:

`  const isAcceleratorRoute = !!useMatch({ path: APP_PATHS.ACCELERATOR, end: false });`

Comment (karthikbtf) [reply src/hooks/useAcceleratorConsole.ts:10]: Nice, let me try that 👍

Comment (karthikbtf) [reply src/hooks/useAcceleratorConsole.ts:10]: That worked, updating PR.

File: src/scenes/AcceleratorRouter/Cohorts/index.tsx [hunk 2]
```diff
@@ -10,10 +10,10 @@ import CohortsListView from './CohortsListView';
...
   return (
     <Routes>
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_EDIT} element={<CohortEditView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_NEW} element={<CohortNewView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS_VIEW} element={<CohortView />} />
-      <Route path={APP_PATHS.ACCELERATOR_COHORTS} element={<CohortsListView />} />
+      <Route path={':cohortId/edit'} element={<CohortEditView />} />
+      <Route path={'new'} element={<CohortNewView />} />
+      <Route path={':cohortId'} element={<CohortView />} />
+      <Route path={''} element={<CohortsListView />} />
```
Comment (constanzauanini) [inline src/scenes/AcceleratorRouter/Cohorts/index.tsx:16]: You can put * here and remove next line

Comment (karthikbtf) [reply src/scenes/AcceleratorRouter/Cohorts/index.tsx:16]: No we need the navigate-to UX so url is updated as well.

Comment (constanzauanini) [reply src/scenes/AcceleratorRouter/Cohorts/index.tsx:16]: Oh, true

File: src/scenes/PlantingSitesRouter/index.tsx [hunk 3]
```diff
@@ -89,9 +89,10 @@ export function PlantingSitesRouter({ reloadTracking }: PlantingSitesProps): JSX
 export function PlantingSitesDraftRouter(): JSX.Element {
   return (
     <Routes>
-      <Route path={'/zone/:zoneId'} element={<PlantingSiteDraftZoneView />} />
+      <Route path={'/:plantingSiteId/zone/:zoneId'} element={<PlantingSiteDraftZoneView />} />
```
Comment (constanzauanini) [inline src/scenes/PlantingSitesRouter/index.tsx:92]: Good catch

**Review by constanzauanini — APPROVED** [review]

File: src/scenes/PlantingSitesRouter/index.tsx (hunk 3 above)
Comment (karthikbtf) [reply src/scenes/PlantingSitesRouter/index.tsx:92]: Thanks for updating all the routes, that was a lot of work!

**Pull request merged**